static/sass/components/_images.scss

```diff
@@ -177,6 +177,7 @@
 
 .image-title h1,
 .image-title h3 {
+    word-break: break-word;
     word-wrap: break-word;
```

Comment:
Remove the `word-wrap` property, it is nonstandard and obsolete.

Reply:
This is probably a good idea — would you mind opening a new ticket for that? Then I can do the proper testing and also update the dozens of other spots in the code where we're using these properties.

Reply:
@spaceninja I'll make one for a general SCSS code cleanup; without looking at the current state of things I don't have a good sense of how aggressive the task should be.

Reply:
#580